### PR TITLE
Add Constants - part 3

### PR DESCRIPTION
**Pull request by bouwew**

### CODE REVIEW

File: custom_components/plugwise/__init__.py [hunk 1]
```diff
@@ -122,12 +122,8 @@ def async_migrate_entity_entry(entry: er.RegistryEntry) -> dict[str, Any] | None
         return {"new_unique_id": entry.unique_id.replace("-slave_boiler_state", "-secondary_boiler_state")}
     if entry.domain == Platform.SWITCH and entry.unique_id.endswith("-plug"):
         return {"new_unique_id": entry.unique_id.replace("-plug", "-relay")}
-    if entry.domain == Platform.SENSOR and entry.unique_id.endswith(
-        "-relative_humidity"
```
Comment (CoMPaTech) [inline custom_components/plugwise/__init__.py:126]: expecting this to be formatting? or would we want to partial the strings in this section as well?

File: custom_components/plugwise/climate.py [hunk 2]
```diff
@@ -250,11 +250,11 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
         data: dict[str, Any] = {}
         if ATTR_TEMPERATURE in kwargs:
-            data[SETPOINT] = kwargs.get(ATTR_TEMPERATURE)
```
Comment (CoMPaTech) [inline custom_components/plugwise/climate.py:253]: setpoint is a pretty industry standard ... or is HA moving to target temp?

File: custom_components/plugwise/const.py [hunk 3]
```diff
@@ -44,16 +44,24 @@
 DEV_CLASS: Final = "dev_class"
 NONE : Final = "None"
 MODE: Final = "mode"
-SETPOINT: Final = "setpoint"
-SETPOINT_HIGH: Final = "setpoint_high"
-SETPOINT_LOW: Final = "setpoint_low"
+TARGET_TEMP: Final = "setpoint"
```
Comment (CoMPaTech) [inline custom_components/plugwise/const.py:47]: see previous comment

File: custom_components/plugwise/const.py [hunk 4]
```diff
@@ -73,7 +81,50 @@
...
 
 # Sensor constants
+DHW_TEMP: Final = "dhw_temperature"
+DHW_SETPOINT: Final = "domestic_hot_water_setpoint"
+EL_CONSUMED: Final = "electricity_consumed"
+EL_CONSUMED_INTERVAL: Final = "electricity_consumed_interval"
+EL_CONSUMED_PEAK_INTERVAL: Final = "electricity_consumed_peak_interval"
+EL_CONSUMED_OFF_PEAK_INTERVAL: Final = "electricity_consumed_off_peak_interval"
+EL_CONSUMED_PEAK_CUMULATIVE: Final = "electricity_consumed_peak_cumulative"
+EL_CONSUMED_PEAK_POINT: Final = "electricity_consumed_peak_point"
```
Comment (CoMPaTech) [inline custom_components/plugwise/const.py:91]: If we are shortening I would suggest improving this (pretty long names to pretty long names seems to defeat the point): `E_CONS_INT, E_CONS_PEAK, E_CONS_NPEAK`? Not sure, but lets make 'm shorter.

And just PH1/PH2/PH3 would do I guess

File: custom_components/plugwise/__init__.py (hunk 1 above)
Comment (bouwew) [reply custom_components/plugwise/__init__.py:126]: This used to be the result of running Black. But the added line was longer and not formatted by Ruff, so I removed this formatting as a test. I didn't get reformatted by Ruff.

File: custom_components/plugwise/climate.py (hunk 2 above)
Comment (bouwew) [reply custom_components/plugwise/climate.py:253]: I've used the HA climate constant "names" but kept our strings.
I'm thinking, lining up the constant-names is less confusing on HA Core level.

File: custom_components/plugwise/const.py (hunk 3 above)
Comment (bouwew) [reply custom_components/plugwise/const.py:47]: As a follow-up to the above comment: that's the nice thing of a Constant, they don't have to be the same :)

File: custom_components/plugwise/const.py (hunk 4 above)
Comment (bouwew) [reply custom_components/plugwise/const.py:91]: I'll look at this later, thanks.

Comment (bouwew) [reply custom_components/plugwise/const.py]: And implemented.

Comment (bouwew) [reply custom_components/plugwise/const.py]: I didn't short the last word of the EL-related constants, when shorted they made less sense.

File: custom_components/plugwise/climate.py (hunk 2 above)
Comment (CoMPaTech) [reply custom_components/plugwise/climate.py:253]: If it's HA than lets go that route

**Review by CoMPaTech — APPROVED** [review]

**Pull request merged**